app/models/appeals/AppealSubmission.scala

```diff
@@ -629,10 +629,10 @@ object AppealSubmission {
       "appealSubmittedBy" -> appealSubmission.appealSubmittedBy,
       "appealInformation" -> parseAppealInformationToJson(appealSubmission.appealInformation)
     ).deepMerge(
-      appealSubmission.agentDetails.fold(
+      appealSubmission.agentReferenceNo.fold(
         Json.obj()
       )(
-        details => Json.obj("agentDetails" -> details)
+        details => Json.obj("agentReferenceNo" -> details)
```

Comment:
Just realised that it's probably best we change this to:
```suggestion
        agentReferenceNo => Json.obj("agentReferenceNo" -> agentReferenceNo)
```
I'll update my PR as well